package.json

```diff
@@ -56,6 +55,9 @@
     "typescript": "^4.5.5",
     "webpack": "^5.68.0"
   },
+  "peerDependencies": {
+    "@kentico/kontent-delivery": "11.x"
```

Comment:
Just add `@kentico/kontent-delivery` latest with ^´notation to dev dependencies

Reply:
added